lib/content-services/src/lib/search/components/search-facet-field/search-facet-field.component.html

```diff
@@ -8,8 +8,9 @@
                 <input matInput placeholder="{{ 'SEARCH.FILTER.ACTIONS.SEARCH' | translate }}"
                        [attr.data-automation-id]="'facet-result-filter-'+field.label" [(ngModel)]="field.buckets.filterText">
                 <button *ngIf="field.buckets.filterText" mat-button matSuffix mat-icon-button
+                        [title]="'SEARCH.FILTER.BUTTONS.CLEAR' | translate"
```

Comment:
One more thing I missed, I think it should be `[attr.title]` not just `[title]` since you want to apply the attribute

Reply:
made the change